dojo/tools/aws_prowler_v3/parser.py

```diff
@@ -17,15 +17,92 @@ def get_label_for_scan_types(self, scan_type):
         return AWSProwlerV3Parser.SCAN_TYPE[0]
 
     def get_description_for_scan_types(self, scan_type):
-        return "Export of AWS Prowler JSON V3 format."
+        return "Exports from AWS Prowler v3 in JSON format or from Prowler v4 in OCSF-JSON format."
```

Comment:
Please also reflect v4 in the SCAN_TYPE in lines 10,11,14,17 and rename this folder maybe from aws_prowler_v3 to aws_prowler_v3_v4
Also add the SCAN_TYPE to settings.dist.py deduplication

Reply:
Hi @manuel-sommer, different scan_types would make it impossible to deduplicate formerly imported findings by prowler v3 with newly imported findings by prowler v4. Hence both parsers need to use the same scan_type.

Reply:
Hi @kagahd, you can rename the scantype from `AWS Prowler V3` to `AWS Prowler V3 and V4` just to reflect for the community that also V4 can be imported here. For exisitng findings, just add a migrations file, like https://github.com/DefectDojo/django-DefectDojo/pull/9355 or https://github.com/DefectDojo/django-DefectDojo/pull/9522. It basically migrates the exisiting findings to the new scantype name.

Reply:
Hi @manuel-sommer, I renamed the parser and its package to `AWSProwlerV3plus` to reflect for the community that besides of prowler v3 reports, also reports of later prowler versions can be imported here. 
Changing the `scan_type` accordingly would be fine but is out of my scope for this contribution because it requires too much time for me to implement which I currently don't have. For my company, it works as it is and I just wanted to share the work with the community without putting still more time in it.
It's a pity that the former PR for the prowler v3 parser was accepted. It would have been better to extend the existing prowler parser instead to introduce a new one which scan_type is/was bound to a specific prowler version.
Feel free to adapt the code to fit DefectDojo's requirements.